src/UserService.Business/Commands/Certificate/CreateCertificateCommand.cs

```diff
@@ -41,7 +41,7 @@ public class CreateCertificateCommand : ICreateCertificateCommand
 
             Guid userId = _httpContextAccessor.HttpContext.GetUserId();
 
-            string errorMessage = "Can not add certificate image to certificate. Please try again later.";
+            const string errorMessage = "Can not add certificate image to certificate. Please try again later.";
```

Comment:
why do you use const here?

Reply:
Because i can
I used const when i could do it, because compilation consts are better.
And with all logMessages.